index.html

```diff
@@ -0,0 +1,142 @@
+    </div>
+    <div class="bottom-nav">
+      <div class="icon">
+        <a href="index.html"><i class="fas fa-code fa-3x"></i>
+          <h1>LL Coding Together</h1>
+        </a>
+        <ul class="links">
+          <li class="link"><a href="about.htm">About</a></li>
+          <li class="link"><a href="">Program</a></li>
+          <li class="link"><a href="">Join</a></li>
+          <li class="link"><a href="">Sponsor</a></li>
+          <li class="link"><a href="">News</a></li>
+          <li class="link"><a id="border" href="">LL Coding</a></li>
+        </ul>
+      </div>
+    </div>
+
+      <i class="fas fa-bars fa-3x"></i>
+      <nav id="menu" class="invisible">
+        <ul class="menu-ul">
+            <li><i class="fal fa-times fa-3x cancel-icon"></i></li>
+            <li><a href="about.htm">About</a></li>
+            <li><a href="#About">Join</a></li>
+            <li><a href="#Contact">Sponsor</a></li>
+            <li><a href="#Contact">News</a></li>
+            <li><a href="#Contact">LL Coding</a></li>
+        </ul>
+    </nav>
+    </nav>
+  </header>
+
+  <main>
+    <section class="section-1">
+      <h2>"Hello World!"</h2>
+      <h1>LET'S LEARN CODING TOGETHER</h1>
+      <p>Platform that consists of an interactive learning web platform, an online community forum, chat rooms, online publications and local organizations that intend to make learning web development accessible to anyone. </p>
+      <article>
+        <h2>2021.10.22(FRI) ~ 30(SAT)</h2>
+        <p>@ Addis Ababa, Ethiopia, Africa.</p>
+      </article>
+
+    </section>
+  </main>
+</div>
+
+  <section class="section-2">
+    <h1>Main Program</h1>
+    <hr>
+    <div class="section-2-container">
+
+    
+    <article>
+      <i class="far fa-podium fa-2x"></i>
+      <h2>Lecture</h2>
+      <p>Listen to the speakers from various background share their experiences and life story.</p>
+    </article>
+    <article>
+      <i class="fab fa-wpforms fa-2x"></i>
+      <h2>Forum</h2>
+      <p>Listen to the speakers from various background share their experiences and life story.</p>
+    </article>
+    <article>
+      <i class="fal fa-stamp fa-2x"></i>
+      <h2>Certificate</h2>
+      <p>Listen to the speakers from various background share their experiences and life story.</p>
+    </article>
+    <article>
+      <i class="far fa-hands-helping fa-2x"></i>
+      <h2>Collaboration</h2>
+      <p>Listen to the speakers from various background share their experiences and life story.</p>
+    </article>
+    <article>
+      <i class="far fa-tasks fa-2x"></i>
+      <h2>Projects</h2>
+      <p>Listen to the speakers from various background share their experiences and life story.</p>
+    </article>
+  </div>
+    <button>Join LL Coding Together</button>
+    <a class="desktop a-link" href="#">SEE THE WHOLE PROGRAM</a>
+
+  </section>
+
+  <section class="section-3">
+    <h1>Featured Speakers</h1>
+    <hr>
+
+    <div class="speaker-container">
```

Comment:
- [ ] Please add this element to your featured speaker's section to comply with the project requirements:

![image](https://user-images.githubusercontent.com/38522578/137224947-24d621c0-ab24-484a-a112-9fc7e40f5fe6.png)

- [ ] Please also set the same size to all the images to comply with the project requirements:

![image](https://user-images.githubusercontent.com/38522578/137225012-343bb49f-6ed2-41b4-8d84-6b91ca08921e.png)